app/models/dossier.rb

```diff
@@ -221,10 +221,12 @@ def classer_sans_suite(motivation: nil, instructeur: nil, processed_at: Time.zon
   scope :prefilled,                 -> { where(prefilled: true) }
   scope :hidden_by_user,            -> { where.not(hidden_by_user_at: nil) }
   scope :hidden_by_administration,  -> { where.not(hidden_by_administration_at: nil) }
-  scope :visible_by_user,           -> { where(for_procedure_preview: false).where(hidden_by_user_at: nil, editing_fork_origin_id: nil) }
+  scope :hidden_by_expired,         -> { where.not(hidden_by_expired_at: nil) }
+  scope :visible_by_user,           -> { where(for_procedure_preview: false).where(hidden_by_user_at: nil, editing_fork_origin_id: nil).where(hidden_by_expired_at: nil) }
```

Comment:
on pourrait alléger l'écriture

```suggestion
  scope :visible_by_user,           -> { where(for_procedure_preview: false, hidden_by_user_at: nil, editing_fork_origin_id: nil, hidden_by_expired_at: nil) }
```